blockly/msg/json/nl.json

```diff
@@ -7,7 +7,8 @@
 			"MedShot",
 			"아라",
 			"JaapDeKleine",
-			"Sjoerddebruin"
+			"Sjoerddebruin",
+			"Benny Malengier"
```

Comment:
I'm not quite sure, but it sounds like this might be generated directly by translatewiki, so might be lost on a future pull request.